components/core/tools/scripts/lib_install/openssl.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+cUsage="Usage: ${BASH_SOURCE[0]} <version> [prefix] [openssldir]"
+if [ "$#" -lt 1 ] ; then
+    echo $cUsage
+    exit
+fi
+version="$1"
+prefix="${2:-/opt/openssl-${version}}"
+openssldir="${3:-$prefix/ssl}"  # Default to prefix/ssl if not specified
```

Comment:
```suggestion
prefix="${2:-/usr/local}"
openssldir="${3:-${prefix}/ssl}"
```
- The default value for `prefix` is misleading. Normally, if users don't specify `prefix`, it should be `/usr/local`. How about we set it to the actual default behavior?
- I don't think these types of inline comments are necessary, lol